Assets/MixedRealityToolkit/Extensions/CameraExtensions.cs

```diff
@@ -36,10 +37,73 @@ public static bool IsInFOV(this Camera camera, Vector3 position)
             Vector3 screenPoint = camera.WorldToViewportPoint(position);
 
             return screenPoint.z >= camera.nearClipPlane && screenPoint.z <= camera.farClipPlane
-                && screenPoint.x >= 0 && screenPoint.x <= 1 
+                && screenPoint.x >= 0 && screenPoint.x <= 1
                 && screenPoint.y >= 0 && screenPoint.y <= 1;
         }
 
+
+        // Help to clear caches when new frame runs
+        static private int lastCalculatedFrame = -1;
+        // Map from grabbable => is the grabbable in FOV for this frame. Cleared every frame
+        private static Dictionary<Collider, bool> colliderCache = new Dictionary<Collider, bool>();
+        // List of corners shared across all sphere pointer query instances --
+        // used to store list of corners for a bounds. Shared and static
+        // to avoid allocating memory each frame
```

Comment:
Remove "Sphere pointer query instances"

// List of corners of bounds box for current collider being processed in IsInFOVConeCached
// Shared and static to avoid allocating additional memory each frame

Reply:
Suggestion: I'd also rename to boundsCornerPoints to just be explicit and easier reading

Reply:
Renamed!